apis/nucleus/src/components/listbox/components/ListBoxSearch.jsx

```diff
@@ -63,10 +63,15 @@ export default function ListBoxSearch({
 
   useEffect(() => {
     if (wildcardOn && inputRef.current) {
-      const cursorPos = value.length - 1;
-      inputRef.current.setSelectionRange(cursorPos, cursorPos); // place the cursor in the wildcard
-      setWildcardOn(false);
+      const timeout = setTimeout(() => {
```

Comment:
If we are not able to fix this in some other way than using a setTimeout and it seem to be a solid solution, let's at least add a short comment explaining why it's needed.